### PR TITLE
Add sites table for PCA

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Add a table of representative sites to conduct PCA on datasets, e.g., within the ancestry stage of the large cohort pipeline.

### CODE REVIEW

**Review by illusional — APPROVED** [review]
I can't review for correctness, but docs look good to me!

**Comment by MattWellie** [comment]
@KatalinaBobowik is this something we're copying once, or regenerating frequently?
I would have thought this section of the references script would be a better model - you can add the HGDP bucket as a source and get CI to copy it over, rather than adding a separate script to do the copy
https://github.com/populationgenomics/references/blob/main/references.py#L197

**Comment by illusional** [comment]
The service-account here probably does not have read access to the hgdp bucket - because it's not public.

**Comment by KatalinaBobowik** [comment]
@MattWellie this is just something that will be copied once.

So to clarify, under `src` I would add where the sites table is copied from (i.e., `gs://cpg-hgdp-1kg-main/sites_table/v1-0/pruned_variants.ht/`), is that right? 

> see that it has an attached gsutil cp -r command

Do I need to add `gsutil cp -r` somewhere? Or add in `transfer_cmd=gcs_cp_r`?

**Comment by MattWellie** [comment]
@illusional I hadn't spotted that - in addition to this change, we need to add `references` to [here](https://github.com/populationgenomics/cpg-infrastructure-private/blob/main/tokens/repository-map.json#L63) in order to run this transfer script?

@KatalinaBobowik based on Michael's comment I'd ignore my suggestion and go with your current approach. Just for reference if it were an accessible bucket this:
```
Source(
    name='ancestry',
    src='gs://cpg-hgdp-1kg-main/sites_table/v1-0',
    dst='ancestry',
    transfer_cmd=gcs_cp_r,
    files=dict(sites_table='pruned_variants.ht'),
)
```

would result in the new config section:
```
[references.ancestry]
sites_table=gs://cpg-common-main/references/ancestry/pruned_variants.ht
```

**Comment by illusional** [comment]
Good shout @MattWellie - need that tokens/repository-map change to run this.

File: references.py [hunk 1]
```diff
@@ -338,5 +338,14 @@ def is_folder(self) -> bool:
             fasta='hg38/hg38.fa'
         ),
     ),
+    Source(
+        'representative_sites_table',
```
Comment (MattWellie) [inline references.py:342]:
```suggestion
        'ancestry',
```

If you want the simple config entry
```
[references.ancestry]
sites_table=gs://cpg-common-main/references/ancestry/pruned_variants.ht
```
I think this value should also be `ancestry`

**Review by MattWellie — APPROVED** [review]
looks good, just double check the config outcome will be as expected

**Comment by KatalinaBobowik** [comment]
Thanks for that suggestion @MattWellie , I've updated it now 👌

**Pull request merged**